oraclelinux7/scripts/build-rhsm.sh

```diff
@@ -1,18 +1,24 @@
 #!/bin/bash
 #
-# Copyright (c) 2021 Avi Miller
+# Copyright (c) 2021, 2023 Avi Miller
```

Comment:
It's 2024, not 2023. :) Also, you'll want to add another copyright line for yourself, not modify mine.